### PR TITLE
feat: add vertical property at PDivider component

### PR DESCRIPTION
**Pull request by minseolee**
## To Reviewers
- [ ] Skipping reviews is not a problem

## Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

## Description
Add vertical property at PDivider component.
The origin, the PDivider component creates only horizontal divider.
When developers need to create vertical divider component, they have to draw it by css selector such as :after.
So I've added the property, and also changed code structures because the code looks like old.

## Things to Talk About
This feature is unfit to semantic web.
It could be a experimental feature.

### CODE REVIEW

File: src/layouts/divider/PDivider.vue [hunk 1]
```diff
@@ -1,30 +1,41 @@
-<script lang="ts">
-import { getBindClass } from '@/util/functional-helpers';
+<template>
+    <div v-if="vertical" class="p-vertical-divider" />
+    <hr v-else class="p-divider">
```
Comment (WANZARGEN) [inline src/layouts/divider/PDivider.vue:3]: `fragment` 없이 && root element로 감싸지 않고 문제없이 동작하나요?

Comment (minseolee) [reply src/layouts/divider/PDivider.vue:3]: designSystem storybook 에서는 문제 없었습니다. console 에서는 사이드이펙트 고려 못 했습니다. 잘못되거나 누락된 코드가 있을까요?

File: src/layouts/divider/PDivider.vue [hunk 2]
```diff
@@ -1,30 +1,41 @@
-<script lang="ts">
-import { getBindClass } from '@/util/functional-helpers';
+<template>
+    <div v-if="vertical" class="p-vertical-divider" />
```
Comment (WANZARGEN) [inline src/layouts/divider/PDivider.vue:2]: The component's class name should be the kebab case of that component name. Considering the case where you need to customize the css on the page, we set the rules like this.

```suggestion
    <component :is="vertical ? 'div' : hr" class="p-divider" :class="{vertical}" />
```

File: src/layouts/divider/PDivider.vue (hunk 1 above)
Comment (WANZARGEN) [reply src/layouts/divider/PDivider.vue]: Oh, I didn't know, so I asked. There seems to be no problem with writing like this!

File: src/layouts/divider/PDivider.vue [hunk 3]
```diff
@@ -1,30 +1,36 @@
-<script lang="ts">
-import { getBindClass } from '@/util/functional-helpers';
+<template functional>
+    <component
+        :is="props.vertical ? 'div' : 'hr'"
+        :class="[{'p-divider': true}, {'vertical': props.vertical}]"
```
Comment (WANZARGEN) [inline src/layouts/divider/PDivider.vue:4]: In the template based functional syntax, is the way of accessing to props different?

**Review by WANZARGEN — APPROVED** [review]
Looks good to me!

File: src/layouts/divider/PDivider.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (WANZARGEN) [reply src/layouts/divider/PDivider.vue:4]: Okay. I don't think it goes against convention. 😄

**Pull request merged**